docs/rules/no-empty-pattern.md

```diff
@@ -0,0 +1,42 @@
+# No empty pattern (no-empty-pattern)
+
```

Comment:
@ilyavolodin what about this?:

"ECMAScript2015 supports destructuring, to extract data from objects and arrays, and also to handle function parameters. This rule tries to prevent a common mistake where the developer intent is to provide a default value, but creates an empty pattern instead."